### PR TITLE
Add LTX 2.3 Image Model Generation Support

### PR DESCRIPTION
**Pull request by elismasilva**
## Summary

This PR adds native ComfyUI support for image-only LTXV 2.3 checkpoints while keeping the existing LTXV video path on the current `LTXVModel` implementation.

The new image path is routed through a dedicated `LTXVImage` / `LTXVImageModel` pair and is selected only when checkpoint metadata marks the model as image-only.

## What Changed

- Added optional gated attention support to LTXV transformer blocks.
- Made the LTX base `attention_mask` argument optional for workflows that do not provide one.
- Added per-block STG perturbation routing for LTXV self-attention.
- Added `LTXVImageModel` for image-only LTXV checkpoints with image-specific text embedding preprocessing.
- Added `LTXVImage` to supported model detection using the `ltxv_image` model id.
- Added explicit checkpoint metadata handling for the `ltxv_image` model id.
- Added `LTXVGuidanceRescale`, matching Diffusers-style guidance rescale behavior.
- Added LTX 2.3 Image Dev and Distilled text-to-image blueprints.

## Video Compatibility

The existing LTXV video path continues to instantiate `LTXVModel` through `model_base.LTXV`.

Image-only behavior is isolated behind:

- checkpoint metadata: `image_model="ltxv_image"`
- model id: `image_model="ltxv_image"`
- model class: `LTXVImageModel`

The shared LTXV changes are default-off or opt-in:

- `apply_gated_attention` defaults to `False`.
- `attention_mask` now defaults to `None` but existing callers passing a mask are unchanged.
- STG block routing only activates when `transformer_options["stg_self_attn_blocks"]` is set by the STG node.
- `LTXVGuidanceRescale` is a new node and does not affect existing workflows unless connected.

## New Blueprints

- `blueprints/Text to Image (LTX-2.3 Image Dev).json`
- `blueprints/Text to Image (LTX-2.3 Image Distilled).json`

Both are registered under:

`Image generation and editing/Text to image`

The Dev blueprint exposes:

- prompt
- negative prompt
- width / height / duration
- checkpoint
- text encoder
- steps
- CFG
- PAG scale / blocks
- guidance rescale
- optional distilled LoRA and strength

The Distilled blueprint exposes the distilled image workflow with CFG-free defaults and PAG/guidance-rescale controls.

## Suggested Commit Review Order

1. `Add optional gated attention for LTXV blocks`
2. `Make LTX attention mask optional`
3. `Route LTXV STG self-attention perturbations per block`
4. `Add LTXV image-only model support`
5. `Add LTXV guidance rescale node`
6. `Add LTX 2.3 image generation blueprints`

## Validation

Local checks performed:

- Compiled the modified Python files with `python -B` / `compile(...)`.
- Ran `git diff --check` successfully.
- Verified the image-only Distilled blueprint locally in ComfyUI.
- Verified that the Dev blueprint exposes negative prompt, CFG, steps, PAG, guidance rescale, and optional distilled LoRA inputs.

Recommended manual validation before merge:

- Run the existing native `Text to Video (LTX-2.3)` blueprint to confirm the video path remains unchanged.
- Run both new image blueprints with image-only pruned checkpoints.
- Test Dev with distilled LoRA strength `0.0` and with a non-zero strength.
- Test external BF16 text encoder and quantized text encoder variants.

## Checkpoints

The image-only checkpoints used for validation are available at:

- https://huggingface.co/elismasilva/ltx2.3-image-comfyui

## Notes

The image-only checkpoint is expected to include metadata with `image_model="ltxv_image"`. The detection is intentionally explicit so existing LTXV/LTXAV video checkpoints are not rerouted by image-only heuristics.

## Examples generated
image

image

image

## Others
This already working on diffusers:
- https://huggingface.co/spaces/elismasilva/ltx2.3-image-distilled

### CODE REVIEW

File: blueprints/Text to Image (LTX-2.3 Image Dev).json [hunk 1]
```diff
@@ -0,0 +1,2607 @@
...
+            ],
+            "properties": {
+              "cnr_id": "comfy-core",
+              "ver": "0.3.75",
+              "Node name for S&R": "LoraLoaderModelOnly"
+            },
+            "widgets_values": [
+              "ltx-2.3-22b-distilled-lora-384.safetensors",
+              0.0
+            ]
```
[the remark this answers is not in the document]
Comment (elismasilva) [reply blueprints/Text to Image (LTX-2.3 Image Dev).json]: Resolved in `1780f9ec`.

There is currently no compatible 13B image LoRA asset for this checkpoint, so I removed the incompatible 22B distilled LoRA from the Image Dev blueprint instead of declaring it as a model dependency.

The Dev blueprint now uses the 13B image-dev checkpoint directly and exposes `steps`, `cfg`, and `negative_prompt` for non-distilled sampling. Distilled usage is covered by the separate Image Distilled blueprint/checkpoint.
